src/tests/test_filesystem.py

```diff
@@ -105,3 +105,146 @@ def test_idbfs_persist_code(selenium_standalone):
     )
     # remove file
     selenium.run_js(f"""pyodide.FS.unlink("{mount_dir}/test_idbfs/__init__.py")""")
+
+
+@pytest.mark.xfail_browsers(
+    node="Not available", firefox="Not available", safari="Not available"
+)
+def test_nativefs_dir(request, selenium_standalone):
+
+    if request.config.option.runner == "playwright":
+        pytest.xfail("Playwright doesn't support file system access APIs")
+
+    selenium = selenium_standalone
```

Comment:
Should these tests check whether the file appears on the local file system?

Reply:
Yes it has tests about that: `dirHandleMount.entries()` will list files in the local file system and I added a check that files generated from Python(Pyodide) are visible in  `dirHandleMount.entries()`

Reply:
Shouldn't you be able to see them in the actual file system outside of selenium too? Also how do these tests handle the directory picker?

Reply:
That's a good point. Actually, the test in this PR uses [OPFS (Origin Private File System)](https://webkit.org/blog/12257/the-file-system-access-api-with-origin-private-file-system/) which uses IndexDB not a real local file system.

I wasn't able to add a test with *real* local file system, using `showDirectoryPicker()` API. That API requires user gesture and I wasn't able to find a way to test it using selenium.

Reply:
Great could you add a comment saying this to the test? Then I'm +1 to merge it.